Modules/FindNetCDF.cmake

```diff
@@ -231,6 +243,10 @@ foreach( _comp IN LISTS _search_components )
       if( NOT _comp MATCHES "^(C)$" )
         target_link_libraries(NetCDF::NetCDF_${_comp} INTERFACE NetCDF::NetCDF_C)
       endif()
+      if(NetCDF_PARALLEL)
+	find_package(MPI REQUIRED COMPONENTS ${_comp})
```

Comment:
Does this need to be in a for loop?

Reply:
I suppose it probably only needs to link the C MPI library

Reply:
I did it in the loop because that's the easiest place to access the NetCDF targets since you're already looping over them.

Reply:
I took find MPI out of the loop and put it where it detects parallel